lib/configure.js

```diff
@@ -299,6 +335,25 @@ function configure (gyp, argv, callback) {
       argv.push('-I', config)
     })
 
+    // for AIX we need to set up the path to the exp file
+    // which contains the symbols needed for linking.
+    // The file will either be in one of the following
+    // depeding on whether are are in installed or
+    // development environment:
+    //  - the include/node directory
+    //  - the out/Release directory
+    var node_exp_file = "";
+    if (process.platform == 'aix') {
+      var node_root_dir = findNodeRootDirectory();
+      node_exp_file = path.resolve(node_root_dir, 'include/node/node.exp')
```

Comment:
can you confirm these are included in the header tarballs created upstream with your recent changes?